build/server-image/kube-apiserver/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BASEIMAGE
```

Comment:
TODO: to move the set-cap image to a pre-built image in k8s.gcr.io and then stop building it during quick-release.

Reply:
https://github.com/kubernetes/release/pull/1684#issuecomment-775714439

Reply:
Done! https://github.com/kubernetes/k8s.io/pull/1644.